BasiliskII/src/Unix/ether_unix.cpp

```diff
@@ -273,8 +273,8 @@ bool ether_init(void)
 	}
 #endif
 	else {
-		net_if_type = NET_IF_SHEEPNET;
-		printf("selected Ethernet device type sheep_net\n");
+		net_if_type = NET_IF_VDE;
+		printf("selected Ethernet device type vde\n");
```

Comment:
Nit: This message is similar to line 272 but with different capitalization.

Let's maybe rephrase it to mention this is the default (e.g. add " (default)" at the end).

Also, can you add a comment about why we default to VDE? I don't actually know what makes it a better default than sheep_net.

Also, how does it work if HAVE_LIBVDEPLUG is not true?

.. Or was this debugging code that should be removed?